src/htrun/host_tests_runner/host_test.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, options):
         """ ctor
         """
         HostTestResults.__init__(self)
-        self.mbed = Mbed(options)
+        self.target = Base(options)
```

Comment:
I'm not sure what `Base` does. Surely there's a better name we can use than `Base`?

Reply:
I renamed `Base` to `TargetBase`